pyproject.toml

```diff
@@ -22,12 +22,14 @@ dependencies = ["numba>=0.59.1"]
 
 [project.optional-dependencies]
 cu11 = [
+    "cuda-bindings[cu11]",
```

Comment:
`cu11` doesn't exist as an optional dependency of the `cuda-bindings` package. There's the `all` optional dependency you can use that includes `nvcc`, `nvrtc`, and `nvjitlink` if that's what you're looking for? Otherwise you can use the same `all` optional dependency on the `cuda-python` package and it will pull `cuda-bindings[all]`.